wdn/templates_4.1/scripts/navigation.js

```diff
@@ -665,6 +665,10 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 						switchSiteNavigation($homepageCrumbLink[0], false);
 					});
 
+					$(navSel + ' a').last().focusout(function(){
```

Comment:
This will probably work for a majority of the cases, but it will fail with the navigation switcher (and probably the other event listeners that are bound the individual elements). It could be made better by binding the event to a "safe" parent element and only responding when the trigger matches the expectation. Make sense?